src/Aspire.Hosting.Azure/Aspire.Hosting.Azure.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
     <IsPackable>true</IsPackable>
-    <PackageTags>aspire hosting azure</PackageTags>
+    <PackageTags>aspire integration hosting azure</PackageTags>
```

Comment:
This one doesn't really fall into the "integration" category, does it? It is more the "base Azure" library, but doesn't have any Azure resources in it.

Reply:
Ah yep, i think I thought it was a meta-package. So folks will never reference this directly?

Reply:
It has methods like `AddBicepTemplate` so this can be used standalone, which in my mind makes it an Azure integration package. Thoughts?

Reply:
The issue is that all Azure hosting packages reference this one. So it will always be the highest Azure package in the list (sorted by downloads). And today it is the highest package outside of the core Aspire Hosting packages that are already referenced in the AppHost.

![image](https://github.com/user-attachments/assets/0c914ebb-5836-4197-a2f4-75ef692f1917)

So I think we should take away `integration` here so it doesn't show up in the list. We want new users to pick the concrete Azure types, not the base class.

Reply:
OK fair enough, will update.